WIP: Expose channels through kops cli (#14072)
kubernetes/kops: docs/releases/1.26-NOTES.md

* Control-plane instance groups in new clusters are now created with names starting
with "control-plane-". The names of groups for existing clusters are unchanged.

> * The channels CLI that kOps use to manage addons is now bundled with the kOps binary. The interface is considered unstable and experimental, but useful for addon diagnostics and troubleshooting.

Reviewer: ```suggestion
* The channels CLI that kOps use to manage addons is now bundled with the kOps binary. This is useful for addons diagnostics and troubleshooting.
```
